mesonbuild/minstall.py

```diff
@@ -441,6 +446,25 @@ def do_copyfile(self, from_file: str, to_file: str,
         append_to_log(self.lf, to_file)
         return True
 
+    def do_symlink(self, target: str, link: str, full_dst_dir: str) -> bool:
```

Comment:
Make the arguments pathlib.Path objects instead of strings where appropriate.

Reply:
The entire file uses strings and os.path methods, so I would hardly consider this a PR blocker.